scripts/aggregate-translations/aggregate-translations-cli.js

```diff
@@ -25,13 +25,15 @@ commander
   .option('-d, --directories [directories]', 'Regex pattern to glob search for translations', addCustomDirectory)
   .option('-l, --locales <locales>', 'The list of locale codes aggregate on and combine into a single, respective translation file ', localeList, supportedLocales)
   .option('-o, --outputDir [outputDir]', 'The output location of the generated configuration file', './aggregated-translations')
+  .option('-c, --config [baseDir]', 'The path to pull terra i18n config from')
   .parse(process.argv);
 
 const aggregationOption = {
   baseDirectory: commander.baseDir,
   directories: customSearchDirectories,
   locales: commander.locales,
   outputDir: commander.outputDir,
+  configPath: commander.config,
```

Comment:
Looks like we are missing the aggregate-translations documentation in toolkit, but this option should be added with an example of what a config file would look like.

Reply:
https://github.com/cerner/terra-toolkit/issues/82